packages/create-jazz-app/src/index.ts

```diff
@@ -307,7 +307,7 @@ async function scaffoldProject({
         const metroConfig = `const { getDefaultConfig } = require("expo/metro-config");
 const config = getDefaultConfig(__dirname);
 
-config.resolver.sourceExts = ["mjs", "js", "json", "ts", "tsx"];
```

Comment:
Adding cjs here sounded reasonable to me given that Metro handles both ES and CJS modules, but I'm no RN expert